### ISSUE TITLE
About learning rate

### ISSUE THREAD

**Issue by w2kun** [ShuweiShao/IEBins#15]
Thank you for sharing your code! In your paper, you said the model is trained for 20 epochs, but the code sets it to 50. Which configuration did you use to get the results in your paper?

**Comment by ShuweiShao** [comment]
Hi，the model has converged after training for 20 epochs, but training for more epochs may give better results.  If you find that the accuracy barely changes, you can stop it early. According to our observation, it is difficult to further improve the accuracy when the epoch is greater than 25 or 30.

**Comment by w2kun** [comment]
I see. Thanks for your prompt reply! I am closing this issue.
